packages/mobx/src/types/decorator_fills.ts

```diff
@@ -0,0 +1,33 @@
+// Hopefully these will be main-lined into Typescipt, but at the moment TS only declares the Contexts
```

Comment:
Q: is this planned / or raised as issue somewhere? In that case nice to leave a link in the comments so that we can track it.

Reply:
Not that I've seen - it was just something that seemed logical that such types would be defined. Doing a search doesn't reveal any plans for such.